Mathlib/Data/Ordmap/Ordnode.lean

```diff
@@ -313,11 +313,11 @@ def All (P : α → Prop) : Ordnode α → Prop
   | node _ l x r => All P l ∧ P x ∧ All P r
 
 instance All.decidable {P : α → Prop} : (t : Ordnode α) → [DecidablePred P] → Decidable (All P t)
-  | nil => decidableTrue
+  | nil => instDecidableTrue
```

Comment:
I personally would prefer if we refer to instance names as little as possible.
Can you use `inferInstance` or `inferInstanceAs (some term)` here and elsewhere?

Reply:
Done.